### PR TITLE
5 Extra Points for 5 correct consecutive answers followed by speed increase.

### PR DESCRIPTION
**Pull request by ghost**
5 extra Points for 5 correct consecutive answers.

5 Extra points for correct 5 consecutive answers followed by speed increase giving motive to play extra mile always to keep stacking bonus points.

### CODE REVIEW

**Comment by ghost** [comment]
Kindly see @walterbender @chimosky.

**Comment by chimosky** [comment]
Your changes only show a score increase and no speed increase, your PR title states otherwise, I'm guessing you dropped the speed part.

Did you test this with a kid to see if it'll **"give motive to play extra mile always to keep stacking bonus points."**

**Comment by ghost** [comment]
Didnt tried it with a kid and i think the code logic is made in such a way that more your score higher the speed is so if there is 5 points of increase, suddenly there will also be speed increase you can try playing the activity once with changes.

**Comment by ghost** [comment]
My point being making it fun because more multitudes with more challenges a game has 7-12 years try to engage more again not sure based on my experience only and some articles and research.

**Comment by ghost** [comment]
Kindly see this @quozl if these addition make sense to you?

**Comment by quozl** [comment]
Reviewed.  There's no indication of the bonus?

The commit message does not follow our conventions, I had to infer the intent by looking at the change.

**Comment by ghost** [comment]
Added Bonus Prompt before speeding up, decrease speed also so there can be more progress.

**Comment by quozl** [comment]
Thanks.

**Comment by chimosky** [comment]
You could improve on your commit messages; 
```bash
Indicate Bonus Prompt, Adjust Speed.

Remove white screen

Remove White Screen

Adjust Speed

Adjust Speed

Adjust Speed

Adjust Speed
```

isn't a great commit message as it's not really clear why there's a speed adjustment, it'll have to be inferred by looking at the code.

**Comment by ghost** [comment]
Yes, got it let me try again.

**Comment by ghost** [comment]
Kindly review the changes and commit messages @chimosky @quozl  @walterbender.

**Comment by quozl** [comment]
Commit message is still ugly, I'll reformat and rephrase.
